src/main/java/de/chojo/repbot/service/RepBotCachePolicy.java

```diff
@@ -42,6 +42,10 @@ public boolean cacheMember(@NotNull Member member) {
             return true;
         }
 
+        if (member.getVoiceState().inAudioChannel()) {
+            return true;
+        }
+
```

Comment:
This is already been taken care of in `MemberCachePolicy.VOICE` above
```suggestion
```

Reply:
Oh yeah...